apiserver/plane/utils/host.py

```diff
@@ -1,18 +1,18 @@
-# Python imports
-from urllib.parse import urlsplit
-
 # Django imports
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from rest_framework.request import Request
 
+# Module imports
+from plane.utils.ip_address import get_client_ip
 
-def base_host(request, is_admin=False, is_space=False, is_app=False):
+def base_host(request: Request, is_admin: bool = False, is_space: bool = False, is_app: bool = False) -> str:
```

Comment:
@pablohashescobar The request could also be `HttpRequest` object from base `django.http`. So, let's consider both. `request: Request | HttpRequest`